CHANGELOG.md

```diff
@@ -9,6 +9,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+- Add support to the Console to create organization API keys.
+- Add support to the Console to delete organization API keys.
+- Add support to the Console to update organization API keys.
+- Add support to the Console to list organization API keys.
+- Add support to the JS SDK to manage collaborator API keys and rights.
```

Comment:
If I understand https://keepachangelog.com/en/1.0.0/ correctly and we are following it, entries in the "Added" section should not be prefixed with an additional "Add".